data/resources/ui/sidebar.ui

```diff
@@ -23,7 +23,7 @@
     </section>
   </menu>
   <template class="Sidebar" parent="GtkWidget">
-    <property name="width-request">300</property>
+    <property name="width-request">320</property>
```

Comment:
Sidebar width on telegram desktop (at 100% scale) is 300px, so I think it would be more consistent to bump up the size by 10% rather than 6,6%. That might be also an optimal value if we would want to implement fractional scaling for telegrand some time in the future
 
```suggestion
    <property name="width-request">330</property>
```